code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -268,4 +268,38 @@
 	new /obj/item/weapon/personal_inhaler/combat( src )
 	new /obj/item/weapon/reagent_containers/personal_inhaler_cartridge/large/hyperzine( src )
 	new /obj/item/weapon/reagent_containers/personal_inhaler_cartridge/large/inaprovaline( src )
-	new /obj/item/weapon/reagent_containers/personal_inhaler_cartridge/large( src )
\ No newline at end of file
+	new /obj/item/weapon/reagent_containers/personal_inhaler_cartridge/large( src )
+
+
+/obj/item/weapon/storage/box/syndie_kit/liscensetokill
+	name = "Agent Of Intrigue"
+	desc = "The kit used to complete any infiltration mission."
+
+/obj/item/weapon/storage/box/syndie_kit/liscensetokill/fill()
+	..()
+	new /obj/item/clothing/gloves/combat(src)
+	new /obj/item/clothing/shoes/laceup/syndie(src)
+	new /obj/item/clothing/under/suit_jacket/charcoal/syndie(src)
+	new /obj/item/weapon/flame/lighter/zippo/syndielighter(src)
+
+
+/obj/item/weapon/storage/box/syndie_kit/papadov
+	name = ".380 Pistol"
+	desc = "A box containing a small pistol with a suppressor."
+
+/obj/item/weapon/storage/box/syndie_kit/papadov/fill()
+	..()
+	new /obj/item/weapon/gun/projectile/papadov(src)
+	new /obj/item/weapon/silencer(src)
+	new /obj/item/ammo_magazine/c380m(src)
+	new /obj/item/ammo_magazine/c380m(src)
+
+
+/obj/item/weapon/storage/box/syndie_kit/remoteexplosive
+	name = "Remote Explosives"
+	desc = "A box containing a high yield explosive and a detonator disguised as a zippo."
```

Comment:
There is no zippo detonator here.

Reply:
The detonator spawns with the explosive. When you buy the kit it automaticly has the linked detonator with it